engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkRestManager.scala

```diff
@@ -206,21 +206,21 @@ class FlinkRestManager(config: FlinkConfig, modelData: ModelData, mainClassName:
 
   override protected def cancel(job: ProcessState): Future[Unit] = {
     basicRequest
-      .patch(flinkUrl.path("jobs", job.deploymentId.value))
+      .patch(flinkUrl.path("jobs", job.deploymentId.get.value))
```

Comment:
Is it good way..? In fact, is there possible situation when `job.deploymentId` is not defined?

Reply:
:+1: I fixed it